<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LReportParametereDialog</class>
 <widget class="QDialog" name="LReportParametereDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add parametere</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxType">
       <item>
        <property name="text">
         <string>Bool</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Int</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>String</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Date</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Time</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DateTime</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Url</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UInt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LongLong</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ULongLong</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Double</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Char</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Map</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>List</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>StringList</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ByteArray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BitArray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Locale</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rect</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RectF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Size</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SizeF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LineF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Point</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PointF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RegExp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RegularExpression</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hash</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EasingCurve</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Uuid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ModelIndex</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PersistentModelIndex</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Font</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pixmap</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Brush</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Palette</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Image</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Polygon</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Region</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bitmap</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cursor</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>KeySequence</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TextLength</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TextFormat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Matrix</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Transform</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Matrix4x4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vector2D</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vector3D</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vector4D</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Quaternion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PolygonF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Icon</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SizePolicy</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Default value:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditDefValue"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Descript</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPlainTextEdit" name="plainTextEditDescript"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LReportParametereDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
